### ISSUE TITLE
message ... with <call method> - does not trigger error in 702 syntax

### ISSUE THREAD

**Issue by sbcgua** [abaplint/abaplint#1656]
```abap
class lcl_output_excel definition final.
  public section.
  private section.
    methods:
      raise_error       importing io_cx_excel type ref to zcx_excel.
endclass.

class lcl_output_excel implementation.
  method raise_error.
    data lv_dummy type string.
    message e101(00) with io_cx_excel->if_message~get_text( ) into lv_dummy. " <<<<<<<
    " raise exception type zcx_...
  endmethod.
endclass.
```

<details><summary>abaplint.json</summary>
<p>

```json
{
  "global": {
    "files": "/src/**/*.*",
    "skipGeneratedGatewayClasses": true,
    "skipGeneratedPersistentClasses": true,
    "skipGeneratedFunctionGroups": true
  },
  "dependencies": [],
  "syntax": {
    "version": "v702",
    "globalMacros": [],
    "errorNamespace": "^(Z|Y|LT?CL_|TY_|LIF_)"
  },
  "rules": {
    "colon_missing_space": true,
    "commented_code": false,
    "contains_tab": true,
    "description_empty": true,
    "empty_statement": true,
    "exit_or_check": true,
    "exporting": true,
    "global_class": true,
    "identical_form_names": true,
    "inline_data_old_versions": true,
    "line_only_punc": true,
    "message_exists": true,
    "mix_returning": true,
    "parser_error": true,
    "space_before_colon": true,
    "start_at_tab": true,
    "superclass_final": false,
    "whitespace_end": true,

    "class_attribute_names": {
      "exclude": [],
      "ignoreExceptions": true,
      "statics": "^G.?_.*$",
      "instance": "^M.?_.*$",
      "constants": "^C.?_.*$",
      "ignoreLocal": false,
      "ignoreInterfaces": false      
    },
    "functional_writing": {
      "exclude": [],
      "ignoreExceptions": true
    },
    "indentation": {
      "alignTryCatch": false,
      "globalClassSkipFirst": false,
      "ignoreGlobalClassDefinition": false,
      "ignoreGlobalInterface": true,      
      "exclude": [],
      "ignoreExceptions": true
    },
    "line_length": {
      "exclude": [],
      "length": 140
    },
    "keyword_case": {
      "style": "lower",
      "ignoreExceptions": true,
      "ignoreGlobalClassBoundaries": true,
      "ignoreGlobalClassDefinition": false,
      "ignoreFunctionModuleName": false,
      "ignoreGlobalInterface": false,
      "ignoreLowerClassImplmentationStatement": false,
      "ignoreKeywords": []
    },
    "local_class_naming": {
      "exclude": [],
      "local": "^LCL_.*$",
      "test": "^LTCL_.*$",
      "exception": "^LCX_.*$"
    },
    "method_length": {
      "exclude": [],
      "statements": 130,
      "ignoreTestClasses": true,
      "errorWhenEmpty": false
    },
    "method_parameter_names": {
      "exclude": [],
      "ignoreExceptions": true,
      "importing": "^I.?_.*$",
      "returning": "^R.?_.*$",
      "changing": "^C.?_.*$",
      "exporting": "^E.?_.*$",
      "ignoreNames": []
    },
    "nesting": {
      "exclude": [],
      "depth": 5
    },
    "obsolete_statement": {
      "exclude": [],
      "refresh": true,
      "compute": true,
      "add": false,
      "subtract": false,
      "multiply": true,
      "move": true,
      "divide": true,
      "requested": true,
      "occurs": true,
      "setExtended": true,
      "withHeaderLine": true,
      "fieldSymbolStructure": false,
      "load": false,
      "typePools": true
    },
    "sequential_blank": {
      "exclude": [],
      "lines": 4
    }
  }
}
```
</p>
</details>

**Comment by larshp** [comment]
try again, 2.63.15

**Comment by sbcgua** [comment]
:+1: works now
